Analytics/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -496,6 +525,8 @@ - (void)reset
     [self dispatchBackgroundAndWait:^{
         [[NSUserDefaults standardUserDefaults] setValue:nil forKey:SEGUserIdKey];
         [[NSUserDefaults standardUserDefaults] setValue:nil forKey:SEGAnonymousIdKey];
+        [[NSUserDefaults standardUserDefaults] setValue:@[] forKey:SEGQueueKey];
+        [[NSUserDefaults standardUserDefaults] setValue:nil forKey:SEGTraitsKey];
```

Comment:
No particular reason - to be honest, I copy-pasted these lines out of someone else's fork. I can change it.